doc/radosgw/account.rst

```diff
@@ -187,6 +187,43 @@ Alternatively, you may want to create a new account for each existing user. In
 that case, you may want to add the ``--account-root`` option to make each user
 the root user of their account.
 
+Migrating Notification Topics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Account topics are only supported when ``notification_v2`` feature is enabled,
```

Comment:
are supported only when the ``notification_v2`` feature

Reply:
Done.